### PR TITLE
docs: Outreach WG notes for 2019-04-29

### PR DESCRIPTION
**Pull request by sofianguy**
cc @electron/wg-outreach

### CODE REVIEW

File: wg-outreach/meeting-notes/2019-04-29.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+* JD build electron on Windows 10:
+    * https://hackmd.io/s/rJ67Tjc5V
+    * all told, took days including troubleshooting
+* David build electron on Linux:
+    * https://hackmd.io/YJmFdvFrRYGfNgzW74EHSA 
+    * need to organize the build instructions order
+* Season of Docs follow up
+    * decision will be released on April 30, 2019 12:00 UTC 
+* Follow up on action items from [2019-04-01](https://github.com/electron/governance/blob/master/wg-outreach/meeting-notes/2019-04-01.md#action-items) meeting
+* Action Items [Project Board](https://github.com/electron/governance/projects/4)
```
Comment (jdalton) [inline wg-outreach/meeting-notes/2019-04-29.md]: Is the Project Board a new thing?

Comment (sofianguy) [reply wg-outreach/meeting-notes/2019-04-29.md]: Oops, I forgot to talk about that during the meeting. I made a project board to keep track of our ongoing Action Items. Sometimes we forget to revisit items each meeting.

Comment (sofianguy) [reply wg-outreach/meeting-notes/2019-04-29.md]: I'll delete that from this set of meeting notes and put it on next meeting's

Comment (jdalton) [reply wg-outreach/meeting-notes/2019-04-29.md]: Ah okay!

**Pull request merged**